deploy/kubelet-config/03-kubelet-config.yaml

```diff
@@ -3,11 +3,9 @@ kind: KubeletConfig
 metadata:
   name: custom-kubelet
 spec:
-  machineConfigSelector:
+  machineConfigPoolSelector:
     matchLabels:
       managed.openshift.io/custom-kubelet: true
   kubeletConfig:
-    EnforceNodeAllocatable:
       system-reserved:
```

Comment:
This is what you're trying to modify, yes?:
https://github.com/kubernetes/kubernetes/blob/9016740a6ffe91bb29824f80c34087b993903bd6/pkg/kubelet/apis/kubeletconfig/v1beta1/types.go#L618-L628

Reply:
I think so, yes.

Reply:
ok, updated to use a map instead.